ios/Classes/TiNfcNfcAdapterProxy.m

```diff
@@ -112,6 +170,28 @@ - (void)readerSession:(NFCNDEFReaderSession *)session didInvalidateWithError:(NS
       NO);
 }
 
+#pragma mark NFCReaderSessionDelegate
+
+- (void)tagReaderSession:(NFCTagReaderSession *)session didInvalidateWithError:(NSError *)error
+{
+  [self fireEvent:@"didInvalidateWithError" withObject:@{ @"error" : error.localizedDescription, @"cancelled" : @(error.code == 200) }];
```

Comment:
same here, before calling event, you should also check if event is set or not.

Reply:
Updated accordingly .